packages/@okta/vuepress-site/docs/guides/build-sso-integration/before-you-begin/openidconnect/prep.md

```diff
@@ -9,7 +9,7 @@ Before you create a new OIDC integration in Okta:
    * A single page app (SPA) is a web app that is contained on a single web page. All code is retrieved when the page is loaded initially - the page does not reload or refresh. A SPA app cannot keep running on a server.
 -->
 
-1. Determine the login and optional logout redirect URIs on your system. A redirect URI is where Okta sends the authentication response and ID token. You can specify more than one URI if required.
+1. Determine the login and (optional) logout redirect URIs on your system. A redirect URI is where Okta sends the authentication response and ID token. You can specify more than one URI if required.
```

Comment:
I called these URIs login and logout because that's what they are in the UI, even though I'm not referencing the specific UI elements here, so that the reader knows to connect the two when they are referenced. Should I put them as "... the sign-in (login) and (optional) sign out (logout) redirect URIs..."?

Reply:
ugh no, please don't do that. haha

just skip those review comments then. makes sense to me.